homeassistant/components/mysensors/sensor.py

```diff
@@ -125,6 +126,7 @@
     "V_FLOW": SensorEntityDescription(
         key="V_FLOW",
         native_unit_of_measurement=LENGTH_METERS,
+        device_class=SensorDeviceClass.DISTANCE,
```

Comment:
I don't think flow is a distance

Reply:
I based it on `native_unit_of_measurement`. Is `native_unit_of_measurement` correct?

Reply:
Note: I have removed the device_class to be on the safe side, but I am still intrigued by the `native_unit_of_measurement`

Reply:
@MartinHjelmare / @functionpointer maybe you can confirm?

Reply:
I think the mysensors docs is incorrect and it should represent a volume per time measurement. Let's not change anything here now.

Reply:
Thanks. This was reverted in 7cb0d675b514795b3f4de0264e44f0c2d5fcee94